src/jobs/keys-update/keys-update.service.ts

```diff
@@ -115,10 +117,17 @@ export class KeysUpdateService {
     // read from database last execution layer data
     const prevElMeta = await this.elMetaStorage.get();
 
+    // handle the situation when the node has fallen behind the service state
     if (prevElMeta && prevElMeta?.blockNumber > currElMeta.number) {
       this.logger.warn('Previous data is newer than current data', prevElMeta);
       return;
     }
+
+    if (prevElMeta?.blockHash && prevElMeta.blockHash === currElMeta.hash) {
+      this.logger.log('Same blockHash, indexing is not required', { prevElMeta, currElMeta });
```

Comment:
i think it is update not indexing . and it will be more clear for someone who read the code

Reply:
done